policyengine_canada/variables/gov/provinces/nb/benefits/nbcb/nb_child_benefit.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_canada.model_api import *
+
+
+class nb_child_benefit(Variable):
+    value_type = int
+    entity = Household
+    label = "New Brunswick child tax benefit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.NB
+
+    def formula(household, period, parameters):
+        income = household("adjusted_family_net_income", period)
+        children = household("nb_child_benefit_eligible_children", period)
+        p = parameters(period).gov.provinces.nb.benefits.nbcb
+        base = p.base * children
+        supplement = household("nb_child_benefit_supplement", period)
+        return (
```

Comment:
Suggested breaking up for clarity.

Also to clarify - can households without children receive the supplement? The current logic suggests they can unless I'm misreading.
```suggestion
        eligible = children > 0
        reduction = where(children > 1, p.multiple_children_phase_out.rate.calc(income), p.one_child_phase_out.rate.calc(income))
        phased_out_base = max_(base - reduction, 0)
        return phased_out_base + supplement
```

Reply:
Yes, the working supplement is applied to families with working income regardless of the number or existence of children.